playground/mocks/data/idp/idx/error-authenticator-webauthn-assertion-validation-failure.json

```diff
@@ -0,0 +1,299 @@
+{
```

Comment:
@TianGan-okta  wondering if we can just clone and add respective messages in test rather than having n json files

Reply:
This is a mock for playground, doesn't hurt to have these mock for dev to play with :) what do you think?

Reply:
I am just worried that there are too many json files and we need to update everything if we need to change anything. Since these are all the same api format I dont think dev would test this often. Dont have a strong opinion here. Just wanted to keep the code base clean.